docs-src/python/how-to-send-a-signal-with-start-in-python.md

```diff
@@ -2,24 +2,24 @@
 id: how-to-send-a-signal-with-start-in-python
 title: How to send a Signal-With-Start in Python
 sidebar_label: Signal-With-Start
-description: Signal-With-Start
-tags:
-  - developer-guide
-  - sdk
-  - python
+description: To Signal-With-Start use the `start_workflow()` method and pass the `start_signal` argument with the name of your Signal.
 ---
 
-To send a Signal-With-Start in Python, use the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) method and pass the `start_signal` argument with the name of your Signal, instead of using a traditional Workflow start.
+To send a Signal-With-Start in Python, use the [`start_workflow()`](https://python.temporal.io/temporalio.client.Client.html#start_workflow) method and pass the `start_signal` argument with the name of your Signal.
```

Comment:
The description is very similar to the start of this document. I would provide a brief explanation of why we'd do this and then get into how to do it.

Reply:
agree.